### DISCUSSION TITLE
Next.js with headless CMS - 404 not found page even if the page is created

### DISCUSSION THREAD

**Discussion by annaczinege** [vercel/next.js#33106] [Help]
I use Next.js and Strapi for creating a website. 

I want to add new landing pages to the website, I want to use ISR to load my data from Strapi but I've found an issue with this:

if I go to my-website.com/abcdef before the page exists, I get a 404,
if I then I create the page with this slug, I still have a 404 forever, even after the 60 seconds (revalidate time),
but If I create a new page with the slug 'abcdef' and then go on this page everything seems fine.

What causes this? Is this related to the ISR data fetch or next.js page routing? Or could it be related to Strapi instead?

**Comment by empato-limited** [comment]
+1
